spec/requests/api/v0/post_manifest_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 describe 'POST manifests' do
+  before(:each) do
+    @current_session = mock_authenticated_session
```

Comment:
also, do we need an authed session in order to create a manifest? was not sure if that was required or not